.github/workflows/lint.yml

```
@@ -0,0 +1,35 @@
+name: Lint
+
+on: [push, pull_request]
+
+env:
+  # Configure pip to cache dependencies and do a user install
+  PIP_NO_CACHE_DIR: false
+  PIP_USER: 1
+  PYTHON_VERSION: 3.8
```

```suggestion
  PYTHON_VERSION: 3.9
```
I think we're keeping this 3.9^ as we will only be teaching 3.9^